### DISCUSSION TITLE
v1.7.0-beta.2

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#182] [Announcements]
- New Device: Adds support for the following iCUE LINK devices: XD5 (Stealth Gray), XD5 (White) (#181).

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.7.0-beta.2'>v1.7.0-beta.2</a>.</em>

**Comment by KooolaNL** [comment]
heelo. im not sure if this is the right place, or if i am doing something wrong myself, i use your plugin and all 6 connected fans work like i want them to :DDDD, only thing is, 4 of the fans, disable their leds as soon as i close icue, when i start fancontrol without icue on (as i should) those fans dont light up, when i then open icue it will work but every now and then they go off and on again on random intervals 

im using openrgb  fancontrolv200 and 1.7.0-beta2

thank

**Reply by EvanMulawski** [reply]
Hi @KooolaNL, which controller you do have? Can you upload the CorsairLink.log file (located in the Fan Control directory)? Thanks!

**Reply by KooolaNL** [reply]
commander core xt https://file.io/BAbcgHi9YCd3
